eclass/qt5-build.eclass

```diff
@@ -183,8 +183,13 @@ qt5-build_src_prepare() {
 		qt5_symlink_tools_to_build_dir
 
 		# Avoid unnecessary qmake recompilations
-		sed -i -re "s|^if true;.*(\[ '\!').*(\"\\\$outpath/bin/qmake\".*)|if \1 -e \2 then|" \
-			configure || die "sed failed (skip qmake bootstrap)"
+		if [[ ${QT5_MINOR_VERSION} -ge 8 ]]; then
+			sed -e "/Creating qmake/i if false; then" -e '/cd "$outpath\/qmake"; "$MAKE" -s/a fi' \
```

Comment:
why not adding `fi` after `echo "Done."`?

also, keep one sed expression per line, so break before the second `-e`, and use single quotes for the first sed expression as well